docs/start/api.query.multi.md

```diff
@@ -0,0 +1,60 @@
+# Multi queries
+
+In a number of applications, it is useful to monitor a number of like-queries at the same time. For instance, we may want to track the balances for a list of accounts we have. The `api.query` interfaces allows this via the `.multi` sub-call.
```

Comment:
What are the advantages of using `api.query.multi` vs making the regular queries we learned previously and using `Promise.all`?

Reply:
Oh, I see:

* `.multi` is for _subscribing_ to multiple queries with a single callback.
* `Promise.all` is for reading the _current state_ of multipe queries.

Reply:
`.multi` is also more effective since it bundles queries into a single RPC call, instead of using one per statement. That is one of the things I was hoping to make clear, but have failed at. Will retry :)